### ISSUE TITLE
   run.log_metric

### ISSUE THREAD

**Issue by sqlshep** [MicrosoftDocs/MachineLearning-Python-Feedback#10]
Best i can tell,    run.log_metric("Accuracy", accuracy) is not a thing.  
From Azure Python Notebook execution. 
AttributeError: 'Run' object has no attribute 'log_metric'

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8d2314c0-1812-e30f-17b6-aa60238c00c9
* Version Independent ID: 63186725-185b-876a-5ea6-5fec581d4af5
* Content: [azureml.core.experiment.Experiment class](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.experiment.experiment?view=azure-ml-py)
* Content Source: [AzureML/docs-ref-autogen/azureml-core/azureml.core.experiment.Experiment.yml](https://github.com/MicrosoftDocs/MachineLearning-Python-pr/blob/live/AzureML/docs-ref-autogen/azureml-core/azureml.core.experiment.Experiment.yml)
* Service: **azure**
* GitHub Login: @j-martens
* Microsoft Alias: **jmartens**

**Comment by hning86** [comment]
Correct. it should be run.log("Accuracy", accuracy). We will fix it.
